spec/models/comment_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  let!(:post) { create(:post, message: "Hello") }
+  subject(:comment) { described_class.new(message: "generic comment", post_id: post.id) }
+
+  it "should be valid" do
+    expect(comment.valid?).to eq true
+  end
+  it "post ID should be present"  do
+    comment.post_id = nil
```

Comment:
use a factory to generate a comment without a post, and use fatory_girl's `#build` to not run validation instead of `#create`